### ISSUE TITLE
rotate_credentials gives warning but does not rotate

### ISSUE THREAD

**Issue by bentterp** [basefarm/aws-session-tool#15]
```
bent@BFW-4M5KM72:~$ rotate_credentials -n -p awsops
WARNING: This user already has two sets of access keys. If you wish to rotate both sets, please use the -t flag - but be aware, that the second set of keys will be displayed here on the screen. More information in the wiki.
bent@BFW-4M5KM72:~$ get_session -c
bent@BFW-4M5KM72:~$ get_session -s 684545
WARNING: Your API key is older than 60 days.
```
The way this warning is written, I would expect that the credentials in the profile be rotated and the other set left alone. But it seems that nothing is changed.

**Comment by bjornrog** [comment]
Suggested solution is to improve the wording of the message so it's clear that noting was done and to add an option to delete the **unused** key (based on the assumption that ~/.aws/credentials contains the **used** key).

**Comment by bjornrog** [comment]
Changed the message from warning to error, and changed the wording of the message.

**Comment by bjornrog** [comment]
Fixed in version 1.6.4.
